docs/nadva/nadva-intro.mdx

```diff
@@ -8,9 +8,9 @@ sidebar_position: 1
 > 'Nadva' — isn't that 'Avdan' backward?
 
 
-*Nadva* is AvdanOS' desktop environment which uses wayland as it's display server. Nadva is planned to be a session [Wayland compositor](https://wayland.freedesktop.org).
+*Nadva* is AvdanOS' desktop environment which uses Wayland as its display server. Nadva is planned to be a session [Wayland compositor](https://wayland.freedesktop.org).
 
 In Avdan's [concept video](https://www.youtube.com/watch?v=tXFEiw1aJTw),
-you can see windows interacting in ways that can reminisce both traditional [tiling](https://en.wikipedia.org/wiki/Tiling_window_manager) window managers, and [floating](https://en.wikipedia.org/wiki/Stacking_window_manager) window managers. There are also [dynamic](https://en.wikipedia.org/wiki/Dynamic_window_manager) window managers which let the user to switch between both tiling and floating modes.
+you can see windows interacting in ways that can reminisce both traditional [tiling](https://en.wikipedia.org/wiki/Tiling_window_manager) window managers, and [floating](https://en.wikipedia.org/wiki/Stacking_window_manager) window managers. There are also [dynamic](https://en.wikipedia.org/wiki/Dynamic_window_manager) window managers which lets the user switch between both tiling and floating modes.
```

Comment:
"let" is better here.

Reply:
I think it should be "lets" Or "allows"

Reply:
I think it should be “let” as the noun before it is in its plural form.

Reply:
"let" is indeed better